### ISSUE TITLE
Implement profile edit page

### ISSUE THREAD

**Issue by emilwihlander** [Dsek-LTH/member-page#35]
Implement a page for editing data related to a member.

- create a UI
- Add endpoint to page (e.g. /member/:id/edit)
- Add a mutation to the backend to update member data (e.g. updateMember(id: ${id}))
- Add link from profile view (depends on #34)

It should only be accessible to either yourself or a member of DWWW.

**Comment by LucasBoberg** [comment]
Duplicate of #141
